javascript/1898-maximum-number-of-removable-characters.js

```diff
@@ -0,0 +1,89 @@
+/**
+ * https://leetcode.com/problems/maximum-number-of-removable-characters/
+ * 
+ * Brute force
+ * Time O(removable.length * s.length) | Space O(1)
+ * @param {string} s
+ * @param {string} p
+ * @param {number[]} removable
+ * @return {number}
+ */
+var maximumRemovals1 = function(s, p, removable) {
+
+    let k = 0;
+    // removable.reverse();
+    s = s.split('');
+    p = p.split('');
+    for(let i = 0; i < removable.length; i++) {
+        s[removable[i]] = -1;
+        if(isSubSet()) {
+            k++;
+        } else {
```

Comment:
Suggestion: Avoid else

```js
if (true) continue;

return ...;
```

Reply:
done

Reply:
Sorry for the delay. I got busy. I have updated the space complexity as suggested. I have removed the _else_ case as well. Please check.

Reply:
@aakhtar3, sorry for the delay. I got busy. I have removed the _else_ case as suggested. Please check.